core/src/test/java/org/kohsuke/stapler/test/JettyTestCase.java

```diff
@@ -44,6 +47,8 @@ protected void setUp() throws Exception {
         server.setHandler(context);
 
         ServerConnector connector = new ServerConnector(server);
+        HttpConfiguration hc = connector.getConnectionFactory(HttpConnectionFactory.class).getHttpConfiguration();
+        hc.setUriCompliance(UriCompliance.LEGACY);
```

Comment:
Consistent with https://github.com/jenkinsci/winstone/blob/9149a903679805e97e46eaf100c89735354d2599/src/main/java/winstone/ServerConnectorBuilder.java#L107=. Needed to avoid some sort of test failure if I recall.